plugins/notifications-backend/src/database/DatabaseNotificationsStore.ts

```diff
@@ -155,6 +169,10 @@ export class DatabaseNotificationsStore implements NotificationsStore {
       query.whereNull('notification.saved');
     } // or match both if undefined
 
+    if (options.minimalSeverity !== undefined) {
+      query.where('notification.severity', '<=', options.minimalSeverity);
```

Comment:
Ah, so this is the reason why it's `tinyint`? Why not filter just by exact match and no need to change the column type?

Reply:
It will be harder to add more, we will need to list all cases here.
The numeric comparison seems more clear to me.

But both cases work, I don't have strong opinion on any of them.

Reply:
IMO it would be sufficient to just be able to filter with `equal` operator for the severity and thus the string column type would be sufficient. This adds extra layer of logic to both front- and backend changing the representation from number to string. I would either stick with strings or with numbers in both ways to keep it simple.

Reply:
Yep, we need to filter on DB layer but do we need such filter to include all severities above specific one or would it be sufficient to just filter with equality? String would work for this just as fine.

Reply:
We do not want to be selective in this case, we want to show the selected severity and higher. If the user selects `normal` then all `critical` will be included as well. 

Such thinking/expectation has parallel to setting log-levels. O we use it for searching based on dates.